components/create_comment/create_comment.jsx

```diff
@@ -293,7 +293,7 @@ export default class CreateComment extends React.PureComponent {
         }
 
         const fasterThanHumanWillClick = 150;
-        const forceFocus = (Date.now() - this.lastBlurAt < fasterThanHumanWillClick);
+        const forceFocus = (Date.now() - this.lastBlurAt) > fasterThanHumanWillClick;
```

Comment:
If I'm understanding the intent here, the idea is to return the focus to the comment box when the user clicks on the `Add Comment` button, unless by chance the user intentionally clicked somewhere else in the mean time.

When this code block was first added, this handler was not an `async` method, and so this code probably ran almost immediately after the actual button click. Now, it seems that this time measurement is highly dependent on the `await this.props.onSubmit()` call on line 285 above.

Is there any reason we don't just unconditionally invoke `focusTextbox` earlier in this method, before doing any `await` calls?

Reply:
I don't see any issue with it. Thanks for helping out!